exporter.py

```diff
@@ -34,12 +31,29 @@ def get_config(config_file):
                 config = yaml.load(f, Loader=yaml.Loader)
         except IOError as e:
             LOG.error("Couldn't open configuration file: " + str(e))
+        if 'collectors' not in config:
```

Comment:
If collectors is given as an empty list, you would continue with `[]`. Then the Exporter would still start but it would be an empty prometheus server. Maybe check length?